src/test/spec/unified_runner/mod.rs

```diff
@@ -65,18 +59,8 @@ pub async fn run_unified_format_test_filtered(
     test_file: TestFile,
     pred: impl Fn(&TestCase) -> bool,
 ) {
-    let version_matches = SPEC_VERSIONS.iter().any(|req| {
```

Comment:
as written, this logic would consider a schema version supported if there was any version in the array >= to it. I noticed this when I brought in the new files using schema version 1.6 and didn't need to change the array here to get them running. 

given that, it seemed simpler and more efficient to switch to just storing our max supported version. 

we could take the approach of checking for exact minor version matches instead (which I think may have been the intended idea here), but I think that also doesn't really solve the problem of determining if a file uses unsupported features -- it's possible that a file could e.g. use features from both schema versions 1.6 and 1.7, and therefore report its schema version as 1.7, and we'd consider it supported despite lacking 1.6 support.